### PR TITLE
Feature/button updates/156582235

### PR DESCRIPTION
**Pull request by amiballo**
-update button styling & search bar styling to match

### CODE REVIEW

**Comment by alexisBimatovs** [comment]
@amiballo back to you

**Comment by amiballo** [comment]
@alexisBimatovs Back to you!

**Comment by amiballo** [comment]
@alexisBimatovs addressed the above as shared in design room & back to you!

File: app/assets/stylesheets/forever_style_guide/modules/_nav-mobile-menu.scss [hunk 1]
```diff
@@ -115,18 +118,24 @@ $mobile-menu-close-btn-height: 40px;
...
         border-color: white;
+        box-shadow: none;
 
         &:hover,
         &:focus,
         &:active,
         &:active:focus {
           color: $color-secondary;
-          box-shadow: none;
+
```
Comment (alexisBimatovs) [inline app/assets/stylesheets/forever_style_guide/modules/_nav-mobile-menu.scss]: extra return

**Pull request merged**